### PR TITLE
login page created

### PR DESCRIPTION
**Pull request by Rockyraven**
created login page
added auth
used mockbee backbee
used redux

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=abcdefghijklmno
```
Comment (Rohitprasad83) [inline .env:1]: .env should not be pushed to gitub

File: src/backend/db/users.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ export const users = [
     lastName: "Balika",
     username: "adarshbalika",
```
Comment (Rohitprasad83) [inline src/backend/db/users.js:13]: changed default user

**Review by Rohitprasad83 — APPROVED** [review]
Added few suggestions, rest looks good.

File: .env (hunk 1 above)
Comment (Rockyraven) [reply .env:1]: ok I will correct it

**Review by Shubhamkr2610 — APPROVED** [review]
looks good to me

**Pull request merged**